service/initialise.go

```diff
@@ -145,7 +145,7 @@ func (e *Init) DoGetKafkaConsumer(ctx context.Context, kafkaCfg *config.KafkaCon
 }
 
 // DoGetS3Client returns a S3Client
-func (e *Init) DoGetS3Client(ctx context.Context, cfg *config.S3Config) (sitemap.S3Uploader, error) {
+func (e *Init) DoGetS3Client(ctx context.Context, cfg *config.S3Config) (sitemap.S3Client, error) {
```

Comment:
The ctx parameter is not used
```suggestion
func (e *Init) DoGetS3Client(cfg *config.S3Config) (sitemap.S3Client, error) {
```